.github/workflows/deploy.yml

```diff
@@ -41,12 +45,12 @@ jobs:
       - id:   select-environments
         if:  github.event.inputs.pr == ''
         uses: DFE-Digital/bat-infrastructure/actions/prepare-environment-matrix@main
```

Comment:
need to add `pen` environment in the above [action](https://github.com/DFE-Digital/bat-infrastructure/blob/main/actions/prepare-environment-matrix/action.yml#L7)